adding moco to CI

anijain2305 opened the pull request:
Fixes #ISSUE_NUMBER


cc @soumith @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

desertfire reviewed (commented):
You will need to run `benchmarks/dynamo/ci_expected_accuracy/update_expected.py` after the first round is done.

anijain2305 commented:
@pytorchbot rebase

anijain2305 commented:
@pytorchbot merge

anijain2305 committed:
adding moco to CI

ghstack-source-id: f52ee3e8a3d8aa174d8083d652116cbe9f639feb
Pull Request resolved: https://github.com/pytorch/pytorch/pull/101776

anijain2305 commented:
@pytorchbot merge

anijain2305 commented:
@pytorchbot merge
